deploy/kubernetes-1.13/pmem-unified-csi-direct.yaml

```diff
@@ -0,0 +1,77 @@
+# Deploys the pmem-csi driver onto exactly one node, running in unified mode.
+# That CSI drivers gets exposed with a NodePort service and the fixed port
+# 30123 for use with csi-test sanity testing.
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: pmem-unified
+spec:
+  type: NodePort
+  selector:
+    app: pmem-csi-unified
+  ports:
+  - port: 10000
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: pmem-csi-unified
+  namespace: default
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: pmem-csi-unified
+  serviceName: pmem-csi-unified
+  template:
+    metadata:
+      labels:
+        app: pmem-csi-unified
+    spec:
+      initContainers:
+      containers:
+      - name: pmem-driver
+        image: 192.168.8.1:5000/pmem-csi-driver:canary
+        imagePullPolicy: Always
+        terminationMessagePath: /tmp/termination-log
+        args:  [ "-v=5",
+                 "-drivername=pmem-csi.intel.com",
+                 "-mode=unified",
+                 "-deviceManager=ndctl",
+                 "-endpoint=$(CSI_ENDPOINT)",
+                 "-nodeid=$(KUBE_NODE_NAME)" ]
+        env:
+        - name: CSI_ENDPOINT
+          value: "tcp://0.0.0.0:10000"
+        - name: KUBE_NODE_NAME
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: spec.nodeName
+        securityContext:
+          privileged: true
+        volumeMounts:
+          # We create temporary staging and target directories under /var/tmp
+          # during csi-sanity testing and must have those visible inside the
+          # container (see test/e2e/storage/sanity.go).
+          - name: var-tmp
+            mountPath: /var/tmp
+          - name: dev-dir
+            mountPath: /dev
+            mountPropagation: Bidirectional
```

Comment:
`Bidirectional` Still required?

Reply:
thanks, good catch, I also found it myself around same times, force pushed deletion.
Once creeping in, this line gets copied around at same speed we find and delete it,
but we get better.